packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -168,6 +168,20 @@ contract OPContractsManager is ISemver {
     /// which is intended to be DELEGATECALLed.
     OPContractsManager internal immutable thisOPCM;
 
+    /// @notice The release string.
+    string private constant RELEASE = "op-contracts/vX.Y.Z";
+
+    /// @notice The address of the upgrade controller.
+    address public immutable upgradeController;
+
+    /// @notice Whether this is a release candidate.
+    bool public isRC = true;
+
+    /// @notice Returns the release string. Appends "-rc" if this is a release candidate.
+    function release() external view returns (string memory) {
```

Comment:
There is already an `l1contractsRelease` value above, we should just modify that.